### PR TITLE
Add basic instructions about how to build the docs

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# data-management
+# Research Data Management
+
+This repo provides the documentation for the research data management processes at the [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693), [University of Tübingen](https://ror.org/03a1kwz48)[Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693).
```
Comment (Steffengreiner) [inline README.md:3]:
```suggestion
This repo provides the documentation for the research data management processes at the [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693), [University of Tübingen](https://ror.org/03a1kwz48) [Quantitative Biology Center](https://ror.org/00v34f693) [<img src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height=18>](https://ror.org/00v34f693).
```
NIT: Without a space it looks cluttered:
image

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+## Documentation
+
+The documentation source files themselves are located in this repository's `./docs` folder. You can use simple [Markdown](https://www.markdownguide.org/) syntax for providing more content.
+
+## Deployment
+
+The deployment of the live side is implemented with Github actions, once a commit is done to the `main` branch.
+
+The live site of the build documentation can be then found on https://qbicsoftware.github.io/research-data-management.
```
Comment (Steffengreiner) [inline README.md:41]: Major NIT: 

I think it would be nicer to have this link on the top of the readme so people can check it out asap 🤔

**Review by Steffengreiner — APPROVED** [review]
Heya @sven1103 it looks awesome! 
I found some very minor nitpicks which would improve the readme even further imo.

**Review by KochTobi — APPROVED** [review]
LGTM!!

**Pull request merged**